index.html

```diff
@@ -423,8 +473,13 @@
         acquiring permission or determining the permission status.
       </p>
       <p>
-        When a permission is revoked, all <a>push subscriptions</a> created with that permission
-        MUST be <a>deactivated</a>.
+        When a permission is revoked, the <a>user agent</a> MAY
```

Comment:
I actually think we found a case where we need to do this too. Let me file a more elaborate issue on it in a bit.